plugins/action_revoke_permissions/src/databaseMethods.ts

```diff
@@ -23,10 +23,15 @@ const databaseMethods = (DBConnect: any) => ({
   revokePermissionFromUserOrg: async (
     username: string,
     org: string | number,
-    permissionNames: string[]
+    permissionNames: string[],
+    isRemovingPermissions: boolean
```

Comment:
```suggestion
    isRemovingPermissions: boolean = true
```
Might be good to add default for now.